lib/tasks/Make.js

```diff
@@ -73,10 +88,15 @@ class Make extends Task {
    * @param   { Object } opt - cli options
    */
   toFile(from, to, opt) {
-    this.encapsulate(
-      from.map((path) => this.parse(path, opt)).join('\n'),
-      opt
-    ).to(to[0])
+    co(function* () {
```

Comment:
@cognitom ok nevermind, I am not used to es6 classes anymore I currently use only objects. Leave it as it is then